src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -1097,6 +1097,10 @@ uct_rc_mlx5_iface_common_tag_recv(uct_rc_mlx5_iface_common_t *iface,
                                          tag_mask,
                                          UCT_RC_MLX5_SRQ_FLAG_TM_CQE_REQ |
                                          UCT_RC_MLX5_SRQ_FLAG_TM_SW_CNT);
+
+    UCT_RC_IFACE_TM_STAT(rc_iface, LIST_ADD);
+    UCT_RC_IFACE_TM_STAT(rc_iface, LIST_SYNC);
```

Comment:
i think it's better to count sync ops, only when they are done individually, and not as piggyback on another operation